src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -259,15 +288,15 @@ class TimeSeriesRelation(config: RelationConfig)(val sqlContext: SQLContext)
     (putItems, postItems).parMapN((_, _) => ())
   }
 
-  def baseTimeSeriesUrl(project: String, version: String = "0.5"): Uri =
+  def baseTimeSeriesUrl(project: String, version: String = "v1"): Uri =
     uri"${baseUrl(project, version, config.baseUrl)}/timeseries"
 
   override def schema: StructType = structType[TimeSeriesItem]
 
   override def toRow(t: TimeSeriesItem): Row = asRow(t)
 
   override def listUrl(): Uri =
-    uri"${config.baseUrl}/api/0.5/projects/${config.project}/timeseries"
+    uri"${config.baseUrl}/api/v1/projects/${config.project}/timeseries"
```

Comment:
Looks like we could use `baseTimeSeriesUrl(config.project, "v1")` here as well?

Reply:
We can! Nice.